### ISSUE TITLE
depth value seems wrong

### ISSUE THREAD

**Issue by GOBish** [samhoff20/Realsense-Object-Detection-Public#6]
For some reason the depth value returned seems wrong when using this method.
Strange because I run realsense-viewer and when you hover the mouse over a point in an image, the depth value is correct.

Have you seen this as well?

**Comment by GOBish** [comment]
I think I may have figured it out -
I think you have possibly reversed the X and Y coordinates for getting the depth.
you have:

depth = depth_image[centerX, centerY].astype(float)

It possibly should be:

depth = depth_image[centerY, centerX].astype(float)

**Comment by samhoff20** [comment]
I have noticed that, I thought it might be related to the differing FOV from the color FOV to the aligned FOV. I have not had an opportunity to test your method, did you see positive results. Thanks for pointing this out by the way, I very much appreciate it! Open-source ftw!

**Comment by GOBish** [comment]
I was getting the same results with aligned FOV or not before I made the switch of X and Y.
With the X and Y switch, results are looking very good!
I am pretty sure that was the issue because I would get range errors of the bounding box of detected items before I made the switch. Some close investigation of the X and Y values led me to the conclusion for the switch, plus looking at some other folks' code using depth_image.
